### DISCUSSION TITLE
RbacPlugin

### DISCUSSION THREAD

**Discussion by NixFull** [Sylius/Sylius#12435] [Q&A] [answered]
RbacPlugin is not compatible with sylius V1.9

Someone can help please I'm novice in symfony

The link : https://github.com/Sylius/RbacPlugin

**Comment by vvasiloi** [comment, marked as answer]
That plugin is abandoned, try this fork: https://github.com/odiseoteam/RbacPlugin

**Comment by Arminek** [comment]
As @vvasiloi said, this plugin is not supported anymore.
